src/zenml/utils/tag_utils.py

```diff
@@ -0,0 +1,85 @@
+#  Copyright (c) ZenML GmbH 2022. All Rights Reserved.
```

Comment:
The functionality in this file feels more like it belongs under the `zenml.zen_stores` module than here, maybe even in the `sql_zen_store` module itself. The functions that take `SqlZenStore` as an argument in particular are a good fit for methods under that class.

Reply:
Yeah, agree, makes sense - moved it around.